examples/js/getting-started/package.json

```diff
@@ -9,6 +9,7 @@
     "lint:fix": "npm run lint -- --fix"
   },
   "dependencies": {
+    "@algolia/client-composition": "0.0.1-alpha.1",
```

Comment:
is there a way to publish this publicly? otherwise it won't be mergeable

Reply:
Not at the moment, I'm afraid.
We might end up with a lite public client but it's still being discussed.

I can remove it from the example package once we decide we are ready to merge.
Hopefully by the time I create a dedicated example this package thing will be resolved

Reply:
yes let's add a dedicated example